extensions/interactions/rule-input-defs.ts

```diff
@@ -265,3 +266,6 @@ export type InteractionRuleInputs = (
   RatioInputRuleInputs|
   SetInputRuleInputs |
   TextInputRuleInputs);
+
+export type InteractionId = (
```

Comment:
any specific reason for defining it inside rule-input-defs? (We have rule-input-defs, answer-input-defs, should we add it inside a new file interaction-defs?[not sure!])

Reply:
I was unsure where to define this, and the rule-input-defs was defining the rules for keys of InteractionSpecsConstants.INTERACTION_SPECS, so I defined it along with them. Should I make a new file?

Reply:
Yes, I also think it makes sense to move it to a separate file.

Reply:
@iamprayush Can you guide me where should I place this file?

Reply:
You can create a new file in this directory itself, just like `rule-input-defs.ts`, `answer-defs.ts`, and `customization-args-defs.ts`.

Reply:
Okay, I'm adding the commit with required changes.